conf/messages.en

```diff
@@ -727,7 +727,7 @@ declaration.declarationHolderRequired.mainText.standard_prelodged_1040 = Authori
 declaration.declarationHolderRequired.mainText.occasional.1 = You indicated you're making a simplifed declaration (occasional use).
 declaration.declarationHolderRequired.mainText.occasional.2 = You do not have to hold any authorisations to do this.
 declaration.declarationHolderRequired.mainText.standard_prelodged_other = You need to add authorisations for your special procedure, unless you are using Authorisation by customs declaration.
-declaration.declarationHolderRequired.inset.para1 = You can still add authorisations that you or other people you've added to this declaration hold, such as:
+declaration.declarationHolderRequired.inset.para1 = You can still add authorisations that you or other people you''ve added to this declaration hold, such as:
```

Comment:
We've had an issue before with these, because design team wanted to have the curly version of it. Maybe double check with design if this is fine?

Reply:
But I think you want the curly version when the text is **enclosed** in quotes